projects/kubernetes/python/README.md

```diff
@@ -0,0 +1,198 @@
+# Running Python Application on Kubernetes
+
+## Kubernetes is an open source platform that offers deployment, maintenance, and scaling features. It simplifies management of containerized Python applications while providing portability, extensibility, and self-healing capabilities.
+
+## Whether Python application are simple or complex, Kubernetes lets you efficiently deploy and scale them seamlessly rolling out new features while limiting resources to only those required.
+
+### Pre-requistes
+
+-Docker
+-Kubectl
+-Sample Python App
+
+## Docker is an open platform to build and ship distributed applications. To install Docker, follow the Docker official documentation.
+
+## To verify that Docker runs your system:
+
+```
+$ docker info
+Containers: 0
+Images: 289
+Storage Driver: aufs
+Root Dir: /var/lib/docker/aufs
+Dirs: 289
+Execution Driver: native-0.2
+Kernel Version: 3.16.0-4-amd64
+Operating System: Debian GNU/Linux 8 (jessie)
+WARNING: No memory limit support
+WARNING: No swap limit support
+```
+
+### kubectl is a command-line interface for executing commands against a Kubernetes cluster. Run the shell script below to install kubectl
```

Comment:
kubectl is a command-line interface for executing commands against a Kubernetes cluster. Run the shell script below to install kubectl